keps/sig-cli/2206-openapi-features-in-kustomize.md

```diff
@@ -0,0 +1,492 @@
+---
```

Comment:
Actually - this is no longer the format we use for KEPs.

Please update the KEP to the new template which is:
- open a corresponding issue in this repo
- the issue number will be the number of this KEP
- move it to appropriate subdirectory and split the file into README.md and metadata in kep.yaml - see

https://github.com/kubernetes/enhancements/tree/master/keps/NNNN-kep-template